src/components/Audiences/processDestinationsFactory.js

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import createDestinations from "../../utils/createDestinations";
+import { isObject, isNonEmptyString, cookie } from "../../utils";
+
+export default ({ destinations, config, logger }) => {
+  const urlDestinations = destinations.reduce((arr, dest) => {
+    if (isObject(dest) && dest.type === "url" && isObject(dest.spec)) {
```

Comment:
By ignoring the `dest` when it's not an object, I'm afraid a bigger problem will go unnoticed for too long. I'd rather a customer come to us saying "Hey, we're getting a 'Cannot read property 'type' of undefined' error" as soon as possible so we can fix it promptly for all our customers. Thoughts?

Reply:
Joe agrees.